okio-fakefilesystem/api/okio-fakefilesystem.api

```diff
@@ -1,8 +1,9 @@
 public final class okio/fakefilesystem/FakeFileSystem : okio/FileSystem {
-	public final field clock Lkotlin/time/Clock;
 	public fun <init> ()V
 	public fun <init> (Lkotlin/time/Clock;)V
 	public synthetic fun <init> (Lkotlin/time/Clock;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public synthetic fun <init> (Lkotlinx/datetime/Clock;)V
+	public synthetic fun <init> (Lkotlinx/datetime/Clock;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
```

Comment:
This is the function I very much want to restore.

https://github.com/square/okio/compare/9db5ee3ad8f0147dff4590b296bff7688b927286...jwilson.0630.dont_crash

Reply:
Without it, lots of downstream tests fail like this:

```
java.lang.NoSuchMethodError: 'void okio.fakefilesystem.FakeFileSystem.<init>(kotlinx.datetime.Clock, int, kotlin.jvm.internal.DefaultConstructorMarker)'
	at com.squareup.wire.SchemaBuilder.<init>(SchemaBuilder.kt:32)
	at app.cash.api.ProguardRulesGeneratorTest.stuff(ProguardRulesGeneratorTest.kt:68)
	```